dpdata/vasp/outcar.py

```
@@ -57,12 +119,24 @@
+    if len(force) == 0:
+        raise ValueError("cannot find forces in OUTCAR block")
+    if len(coord) == 0:
+        raise ValueError("cannot find coordinates in OUTCAR block")
+    if len(cell) == 0:
+        raise ValueError("cannot find cell in OUTCAR block")
+    return True
+
+
 # we assume that the force is printed ...
 def get_frames(fname, begin=0, step=1, ml=False, convergence_check=True):
     fp = open(fname)
```

Ruff (0.11.9)

Use a context manager for opening files

(SIM115)